src/raftstore/store/config.rs

```diff
@@ -107,13 +107,23 @@ pub struct Config {
 impl Default for Config {
     fn default() -> Config {
         let split_size = ReadableSize::mb(coprocessor::config::SPLIT_SIZE_MB);
+
+        let raft_base_tick_interval = ReadableDuration::secs(1);
+        let raft_heartbeat_ticks = 2;
+        let raft_election_timeout_ticks = 10;
+
+        let raft_store_max_leader_lease =
+            raft_base_tick_interval * (raft_election_timeout_ticks - 1) as u32;
+        let max_leader_missing_duration =
+            raft_base_tick_interval * (3 * raft_election_timeout_ticks) as u32;
```

Comment:
An election can happen in `[raft_election_timeout_ticks, 2 * raft_election_timeout_ticks)`. So only two round of election is permitted, which is too small when prevote is not enabled. Besides, this may effect uninitialized peers which can take a long time to work.

Reply:
Any suggestions?